### ISSUE TITLE
Broken with node 12

### ISSUE THREAD

**Issue by jurekbarth** [biotope/biotope-build#192]
Because we're using gulp 3, the biotope/build is broken for node 12 and up. Let's keep an eye on possible solutions in order to get that working again.

For reference: https://github.com/biotope/biotope-boilerplate/issues/99

**Comment by timomayer** [comment]
yes we should consider moving to biotope-build vNext from Tiago before node12 will be the official LTS version starting 2019-10-22 so thats our deadline :)
